pkg/apis/neptune/v1alpha1/jointinferenceservice_types.go

```diff
@@ -27,7 +27,7 @@ type JointInferenceServiceSpec struct {
 type EdgeWorker struct {
 	Model                SmallModel           `json:"model"`
 	NodeName             string               `json:"nodeName"`
-	HardExampleAlgorithm HardExampleAlgorithm `json:"hardExampleAlgorithm"`
+	HardExampleMining HardExampleMining `json:"hardExampleAlgorithm"`
```

Comment:
json:hardExampleAlgorithm needs to be updated

Reply:
has been updated